torch/_dynamo/variables/lists.py

```diff
@@ -883,7 +903,9 @@ def call_method(
         elif name == "__contains__":
             assert len(args) == 1
             assert not kwargs
-            return _listlike_contains_helper(self.items, args[0], tx, options)
+            return _listlike_contains_helper(
```

Comment:
There are other calls to this function IIRC, do we need to change those as well?